client-js/app/elements/elements.html

```diff
@@ -8,6 +8,7 @@
 <link rel="import" href="../bower_components/iron-icons/communication-icons.html">
 <link rel="import" href="../bower_components/iron-icons/iron-icons.html">
 <link rel="import" href="../bower_components/iron-input/iron-input.html">
+<link rel="import" href="../bower_components/iron-list/iron-list.html">
```

Comment:
So this html file is a vestige that we have to get rid of. We're putting imported elements in the file that imports them. Sorry I should make an issue of this I keep forgetting